packages/shared/src/components/profile/devcard/DevCard.tsx

```diff
@@ -85,7 +85,7 @@ export function DevCard({
               alt={`avatar of ${user.name}`}
               className={classNames(
                 '-rotate-3 border-white object-cover',
-                showBorder && (isHorizontal ? 'border-8' : 'border-4'),
```

Comment:
Not sure why we removed this. They have different size of borders from the design.